node_modules/col-core/lib/api.js

```diff
@@ -60,7 +60,7 @@ var init = module.exports.init = function(callback) {
   // guarantee that the system is functioning properly anymore so we kill the process. When running
   // in production, the service script will automatically respawn the instance
   process.on('uncaughtException', function(err) {
-    log.error({'err': err}, 'Uncaught exception was raised, restarting the process');
+    log.error({'err': err, 'stack': err.stack}, 'Uncaught exception was raised, restarting the process');
```

Comment:
For more than a year I was going around assuming there was some fundamental barrier to getting a stack trace when the app crashed. It turns out you just have to ask for it.